src/foam/i18n/LocaleTranslationService.js

```diff
@@ -20,8 +29,23 @@ foam.CLASS({
     {
       name: 'getTranslation',
       javaCode: `
-      System.err.println("*********************************** getTranslation");
-      return "foobar";
+        DAO localeDAO = (DAO) x.get("localeDAO");
+
+        Locale localeEntry = (Locale) localeDAO.find(AND(
+          EQ(Locale.SOURCE, source),
+          EQ(Locale.LOCALE, locale)
```

Comment:
Need to also include the VARIANT here.

Reply:
@kgrgreer check if Locale.Locale and Locale.variant match locale? 

AND(EQ(Locale.SOURCE, source), EQ(Locale.LOCALE, locale), EQ(Locale.VARIANT, locale)) ?